mmrazor/engine/runner/subnet_sampler_loop.py

```diff
@@ -316,8 +318,15 @@ def _check_constraints(self, random_subnet: SupportRandomSubnet) -> bool:
 
         self.model.set_subnet(random_subnet)
         fix_mutable = export_fix_subnet(self.model)
-        flops = FlopsEstimator.get_model_complexity_info(
-            self.model, fix_mutable=fix_mutable, as_strings=False)[0]
+        copied_model = copy.deepcopy(self.model)
+        load_fix_subnet(copied_model, fix_mutable)
+
+        estimator = ResourceEstimator()
```

Comment:
Use estimator_cfg to build ResourceEstimator， and not use `input_shape` as fixed kwargs for `::estimate`.

Reply:
Done.